mkosi/backend.py

```diff
@@ -457,6 +457,7 @@ def run(
     cmdline: List[str],
     check: bool = True,
     delay_interrupt: bool = True,
+    stdin: _FILE = None,
```

Comment:
The two changes in this file are not strictly necessary, since `stdin` will be just picked up via `kwargs` and mypy also doesn't complain. 

I also remember @keszybz wanted to have another go at stdin handling in this function anyway.

Reply:
ah, interesting, makes sense